src/plugins/home/public/application/components/_solutions_panel.scss

```diff
@@ -24,97 +24,72 @@
     }
   }
 
+  .homSolutionsPanel__header {
+    border-radius: $euiBorderRadius 0 0 $euiBorderRadius;
+    color: $euiColorEmptyShade;
+
+    img {
+      position: absolute;
+      width: auto;
+    }
+  }
+
   .homSolutionsPanel__enterpriseSearch {
     .homSolutionsPanel__enterpriseSearchHeader {
       background-color: #017d73;
-      color: $euiColorEmptyShade;
-
-      .homSolutionsPanel__enterpriseSearchTopLeftImage {
-        .euiImage__img {
-          position: absolute;
-          top: 0;
-          left: 0;
-          margin-top: $euiSizeS;
-          height: $euiSizeXL;
-          width: auto;
-        }
+
+      .homSolutionsPanel__enterpriseSearchTopLeftImage img {
```

Comment:
As these panel patterns/images are purely cosmetic, would it be better to use CSS background images instead?

Reply:
Probably, yeah. I did a little consolidating of styles here, but the images need more attention. By re-slicing the images, we could simply the styles further - if the padding was baked in, then we could set all the `right`, `top`, `bottom`, `left` values to `0`.